### PR TITLE
fix: XML Comment

### PR DESCRIPTION
**Pull request by workgroupengineering**
## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

File: src/Avalonia.Base/Platform/ITextShaperImpl.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@ public interface ITextShaperImpl
         /// Shapes the specified region within the text and returns a shaped buffer.
         /// </summary>
         /// <param name="text">The text buffer.</param>
+        /// <param name="length">length of text</param>
```
Comment (robloo) [inline src/Avalonia.Base/Platform/ITextShaperImpl.cs:16]: Capitalization and missing period.

File: src/Avalonia.Base/Rendering/DeferredRenderer.cs [hunk 2]
```diff
@@ -49,6 +49,8 @@ public class DeferredRenderer : RendererBase, IRenderer, IRenderLoopTask, IVisua
         /// </summary>
         /// <param name="root">The control to render.</param>
         /// <param name="renderLoop">The render loop.</param>
+        /// <param name="renderTargetFactory">The target render factory.</param>
+        /// <param name="renderInterface"></param>
```
Comment (robloo) [inline src/Avalonia.Base/Rendering/DeferredRenderer.cs:53]: Missing

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0027496-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0027516-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0027518-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Pull request merged**